packages/builder/src/pages/builder/portal/users/users/_components/EmailTableRenderer.svelte

```diff
@@ -5,7 +5,17 @@
   export let row
 </script>
 
-{value}
+<span class="email">
```

Comment:
Could you add a `title={value}` attribute so that the user can see the full email on hover?

Reply:
Done